scripts/generateOverview.java

```diff
@@ -0,0 +1,130 @@
+/*******************************************************************************
+ * Copyright (c) 2025, 2025 Hannes Wellmann and others.
+ *
+ * This program and the accompanying materials
+ * are made available under the terms of the Eclipse Public License 2.0
+ * which accompanies this distribution, and is available at
+ * https://www.eclipse.org/legal/epl-2.0/
+ *
+ * SPDX-License-Identifier: EPL-2.0
+ *
+ * Contributors:
+ *     Hannes Wellmann - initial API and implementation
+ *******************************************************************************/
+
+final Locale LOCAL = Locale.ENGLISH;
+final DateTimeFormatter LONG = DateTimeFormatter.ofPattern("EEEE, MMMM d, yyyy", LOCAL);
+final DateTimeFormatter SHORT = DateTimeFormatter.ofPattern("MM/dd", LOCAL);
```

Comment:
Personally (as I'm used to European date formats) I find the current format confusing and want to suggest to make it a bit more clear (hopefully for all people of different regions with different common date formats:
```suggestion
final DateTimeFormatter LONG = DateTimeFormatter.ofPattern("EEEE, d MMM yyyy", LOCAL);
final DateTimeFormatter SHORT = DateTimeFormatter.ofPattern("dd MMM", LOCAL);
```

But this can also be a change for later.